hil/src/flash.rs

```diff
@@ -65,16 +74,65 @@ fn extract(path_to_rts: &Utf8Path) -> Result<TempDir> {
     Ok(temp_dir)
 }
 
-fn flash_cmd(variant: FlashVariant, extracted_dir: &Path) -> Result<()> {
+fn generate_random_files(output_dir: &Path) -> Result<()> {
+    // Generate random files in Rust
+    let mut rng = DefaultHasher::new();
+    std::time::SystemTime::now()
+        .duration_since(std::time::UNIX_EPOCH)
+        .unwrap()
+        .as_nanos()
+        .hash(&mut rng);
```

Comment:
use the `rand` crate instead of timestamps. Also, pass the random number generator in as a function argument, to make it possible to test this function.